app/controllers/admin-panel/global-system-message.js

```diff
@@ -0,0 +1,45 @@
+import Controller from '@ember/controller';
+
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+import { INPUT_DEBOUNCE } from 'frontend-lmb/utils/constants';
+
+import { restartableTask, timeout } from 'ember-concurrency';
+
+export default class AdminPanelGlobalSystemMessageController extends Controller {
+  @service store;
+  @service('globalSystemMessage') messageService;
+
+  @tracked message;
+
+  get isDisabled() {
+    return !this.message || this.message.trim === '';
+  }
+
+  @action
+  async toggleIsActive() {
+    const isActive = !this.messageService.isActive;
```

Comment:
now that i read this again, it's kind of hard to understand what is actually happening. Can we make this less convoluted by just updating and saving the record's message on blur of the input and by toggling active state on toggle (and save again there).
We then just never delete the message, but toggle it inactive. We still have to create here if the message doesn't exist yet.